docs/source/getting-started.mdx

```diff
@@ -341,13 +316,15 @@ One of the most important concepts of GraphQL is that clients can choose to quer
 ## Combined example
 
 You can view and fork this complete example on CodeSandbox:
-<!-- TODO: Fix this sandbox link after docs examples is up -->
+<!-- TODO(AS4): Fix this sandbox link after docs examples is up, and make sure it matches the final version -->
 <a href="https://codesandbox.io/s/github/apollographql/docs-examples/tree/RK/add-as4-getting-started/apollo-server/v4/getting-started?fontsize=14&hidenavigation=1&theme=dark">
   <img alt="Edit server-getting-started" src="https://codesandbox.io/static/img/play-codesandbox.svg" />
 </a>
 
 ## Next steps
 
+TODO(AS4): Should we temporarily remove this while we don't have full docs?
```

Comment:
This is a _really good_ question that I went back and forth on. 

On one hand, I do feel like providing these resources is helpful for those looking for more information about the basics of GraphQL servers (particularly the "Schema basics" and "Resolvers" articles).  On the other hand, we could wait and reintroduce these when we know they align with the AS4 syntax, so folks don't get confused. I could go either way, @glasser which would you prefer?

Reply:
I think we should make a change to the top level index.mdx page that's like "as4 is in alpha, we've removed most of the docs that haven't been updated yet, you can trust anything actually here is up to date and otherwise go look at AS3, most stuff that isn't addressed explicitly in the migration guide probably still works"

then idk what to do exactly here: I'd kinda like to avoid broken links though?

Reply:
Oh, the below links won't be broken on the main site -> they'll just lead you to the "main" Apollo Server docset, which would be AS3. So `/apollo-server/schema/schema/` should navigate to `https://www.apollographql.com/docs/apollo-server/schema/schema/`

Reply:
I'll make a PR for adding a note to the top-level index.mdx page now